docs/zodan.md

```diff
@@ -0,0 +1,306 @@
+
+On `n3`:
+
+```sql
+SELECT spock.sync_event();
+
+ sync_event
+------------
+ 0/19F8A58
+(1 row)
+```
+Use the returned value of the sync event when querying the source node; for example,  `spock.sync_event` on `n3` returned `0/19F8A58`, so we use that in the call to `spock.wait_for_sync_event`:
+
+```sql
+CALL spock.wait_for_sync_event(true, 'n3', '0/19F8A58'::pg_lsn, 1200000)
+```
+
+7. Next, use a `spock.sync_event` to confirm transactions from `n2` are caught up on our provider node: 
+
+On `n2`:
+
+```sql
+SELECT spock.sync_event();
+
+ sync_event
+------------
+ 0/1A67F40
+(1 row)
+```
+
+Use the value returned by `spock.sync_event` in the next call to `spock.wait_for_sync_event` on `n1`:
+
+```sql
+CALL spock.wait_for_sync_event(true, 'n2', '0/1A67F40'::pg_lsn, 1200000)
+```
+
+Repeat the sync_event and wait_for_sync_event commands between each replica node in your cluster and the source node to ensure that all data is moved to the source node.
+
+8. Then, to start moving data from our source node (`n1`) to our new node (`n4`), we create an **enabled** subscription named (`sub_n1_n4`) on `n4`:
+
+Note that `synchronize_structure`, `synchronize_data`, and `enabled` are `true`.
+
+```sql
+SELECT spock.sub_create(
+    subscription_name := 'sub_n1_n4',
+    provider_dsn := 'host=127.0.0.1 dbname=inventory port=5432 user=alice password=1safepassword',
+    replication_sets := ARRAY['default', 'default_insert_only', 'ddl_sql'],
+    synchronize_structure := true,
+    synchronize_data := true,
+    forward_origins := '{}'::text[],
+    apply_delay := '0'::interval,
+    force_text_transfer := false,
+    enabled := true
+);
+```
+
+9. Next, we use a `spock.sync_event` to confirm that all of the transactions have been synced from our provider node (`n1`) to our new subscriber node (`n4`):
+
+On `n1`:
+
+```sql
+SELECT spock.sync_event();
+ sync_event
+------------
+ 0/1A7D1E0
+(1 row)
+```
+
+Use the value returned by `spock.sync_event` in the call to `spock.wait_for_sync_event` on `n4`:
+
+```sql
+CALL spock.wait_for_sync_event(true, 'n4', '0/1A7D1E0'::pg_lsn, 1200000)
+```
+
+The last argument in the `spock.wait_for_sync_event` command specifies a timeout value (in our example, 1200000 ms.). You can adjust the timeout to meet your network requirements.
+
+10. Then, on `n4` we check  the `spock.lag_tracker` to confirm that all of the content from `n1` has been replicated:
+
+```sql
+SELECT commit_timestamp FROM spock.lag_tracker WHERE origin_name = 'n1' AND receiver_name = 'n4'
+```
+
+On `n1`, we check for the last LSN that was received; that transaction should also be present on `n4`:
+
+```sql
+WITH lsn_cte AS (SELECT spock.get_lsn_from_commit_ts('spk_inventory_n1_sub_n1_n4', '$n4.commit_timestamp'::timestamp) AS lsn) SELECT pg_replication_slot_advance('spk_inventory_n1_sub_n1_n4', lsn) FROM lsn_cte
```

Comment:
$n4.commit_timestamp'::timestamp must be replaced with the value returned by the previous step.